src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -4223,7 +4223,7 @@ def test_single_stencil_extent(dist_mem, tmpdir):
     output3 = ("      INTEGER(KIND=i_def), intent(in) :: f2_extent\n")
     assert output3 in result
     output4 = (
-        "      INTEGER(KIND=i_def) f2_stencil_size\n"
+        "      INTEGER(KIND=i_def), pointer :: f2_stencil_sizes(:) => null()\n"
```

Comment:
I think it's OK to leave this name as `f2_stencil_size` - it's then more consistent with the naming of the other kernel arguments.

Reply:
I've reverted the name change to ???_stencil_size for everything, as you say it keeps things consistent and any possible naming oddness is kept to the PSylayer.